workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx

```diff
@@ -183,9 +183,14 @@ const AppsTable: React.FC<AppsTableProps> = ({items}) => {
 										</DropDown.Item>
 
 										<DropDown.Item
+											disabled={
+												orderTypeExternalReferenceCode !==
+												OrderType.CLOUD
+											}
```

Comment:
What's the connection between

**Show button when CLOUDAPP** when you are disabling the button?

Reply:
Done!